.github/workflows/codeql-analysis.yml

```diff
@@ -13,16 +13,15 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [main]
+    branches: [ main ]
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: [main]
+    branches: [ main ]
   schedule:
-    - cron: "19 23 * * 2"
+    - cron: '19 23 * * 2'
 
 jobs:
   analyze:
-    if: github.repository == 'kawalcovid19/wargabantuwarga.com'
```

Comment:
Your PR keeps undoing the changes from the previous PRs. I'm wondering how you did it.

Please make sure that there's no changes other than your own changes reflected in this PR. Thanks!